sources/packages/backend/apps/api/src/route-controllers/report/models/report.dto.ts

```diff
@@ -20,3 +20,8 @@ export class ReportsFilterAPIInDTO {
   @JsonMaxSize(JSON_10KB)
   params: ReportFilterParamAPIInDTO;
 }
+
+export class InstitutionReportsFilterAPIInDTO extends ReportsFilterAPIInDTO {
+  @IsNumber()
+  programYear: number;
```

Comment:
I believe that the expectation would be the `programYear` being part of the dynamic property `params` in the already existing `ReportsFilterAPIInDTO`. Is it not already the way that API is accepting start and end dates?